src/EnergyPlus/HVACManager.cc

```diff
@@ -2721,13 +2718,13 @@ void ReportAirHeatBalance(EnergyPlusData &state)
                     znAirRpt.OABalanceHeatGain = -thisZoneHB.MDotCPOA * (thisZoneHB.MAT - zone.OutDryBulbTemp) * TimeStepSysSec * ADSCorrectionFactor;
                 }
                 H2OHtOfVap = Psychrometrics::PsyHgAirFnWTdb(state.dataEnvrn->OutHumRat, zone.OutDryBulbTemp);
-                if (thisZoneHB.ZoneAirHumRat > state.dataEnvrn->OutHumRat) {
-                    znAirRpt.OABalanceLatentLoss = thisZoneHB.MDotOA * (thisZoneHB.ZoneAirHumRat - state.dataEnvrn->OutHumRat) * H2OHtOfVap *
-                                                   TimeStepSysSec * ADSCorrectionFactor;
+                if (thisZoneHB.airHumRat > state.dataEnvrn->OutHumRat) {
+                    znAirRpt.OABalanceLatentLoss =
+                        thisZoneHB.MDotOA * (thisZoneHB.airHumRat - state.dataEnvrn->OutHumRat) * H2OHtOfVap * TimeStepSysSec * ADSCorrectionFactor;
                     znAirRpt.OABalanceLatentGain = 0.0;
                 } else {
-                    znAirRpt.OABalanceLatentGain = thisZoneHB.MDotOA * (state.dataEnvrn->OutHumRat - thisZoneHB.ZoneAirHumRat) * H2OHtOfVap *
-                                                   TimeStepSysSec * ADSCorrectionFactor;
+                    znAirRpt.OABalanceLatentGain =
+                        thisZoneHB.MDotOA * (state.dataEnvrn->OutHumRat - thisZoneHB.airHumRat) * H2OHtOfVap * TimeStepSysSec * ADSCorrectionFactor;
```

Comment:
I see 5 TimeStepSysSec * ADSCorrectionFactor. Could be a local, or does the compiler already basically take care of that? Not here, just sayin.